contracts/Neumark.sol

```diff
@@ -108,6 +115,8 @@ contract Neumark is
         uint amount
     )
         internal
+        acceptAgreement(from)
+        acceptAgreement(to)
```

Comment:
We'll be generating quite a lot of events here. What is our gas cost for simple transfer of Neumarks?

Reply:
I'll benchmark this. LogEvents are pretty cheap, about 500 gas or so (+ 400 for each index).

Reply:
With both log events:

```
        Transfer took 81834 gas (€0.52, 1.2% of limit)
```

without log events:

```
        Transfer took 79368 gas (€0.5, 1.2% of limit)
```

So with overhead, the two events cost us `2466` gas.

Reply:
thanks!